src/reference/asciidoc/smb.adoc

```diff
@@ -122,6 +122,55 @@ public MessageSource<File> smbMessageSource() {
 
 For XML configuration the `<int-smb:inbound-channel-adapter>` component is provided.
 
+==== Configuring with the Java DSL
```

Comment:
Can you also add into docs new components you have introduced in this PR: `SmbStreamingMessageSource` and `SmbOutboundGateway`.
Well, perhaps in a new PR as well.

Reply:
Sure, will create a separate branch and PR for smb.adoc